clustered/client/src/main/java/org/ehcache/clustered/client/internal/store/NoInvalidationServerStoreProxy.java

```diff
@@ -20,21 +20,92 @@
 import org.ehcache.clustered.common.internal.messages.EhcacheEntityResponse;
 import org.ehcache.clustered.common.internal.messages.ServerStoreMessageFactory;
 import org.ehcache.clustered.common.internal.store.Chain;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
 import java.util.concurrent.TimeoutException;
 
 /**
  * Provides client-side access to the services of a {@code ServerStore}.
  */
 class NoInvalidationServerStoreProxy implements ServerStoreProxy {
```

Comment:
Clearly the name of the class no longer fits. Can you update to something like `CommonServerStoreProxy` or something else describing what it does?